lib/p2p/Parser.ts

```diff
@@ -60,20 +60,46 @@ interface Parser {
 
 /** Protocol packet parser */
 class Parser extends EventEmitter {
+  private pending: Buffer[] = [];
+  private waiting: number = 0;
```

Comment:
These names aren't that clear. I think maybe `pendingBuffers` or `pendingPacket` for the first and `remainingBytes` for the second are a couple of suggestions.